modules/ding_debt_easy/lib/Nets/Easy/Order.inc

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Nets\Easy;
+
+/**
+ * Representation of Nets Easy order.
+ */
+class Order {
+
+  private $currency = 'DKK';
+  private $reference = '';
+  private $orderItems = [];
+
+  /**
+   * Get currency for this order.
+   *
+   * @return string
+   *   Return the currency (default: DKK)
+   */
+  public function getCurrency() {
+    return $this->currency;
+  }
+
+  /**
+   * Set currency.
+   *
+   * @param string $currency
+   *   The currency to use (should be a three letter string)
+   *
+   * @return $this
+   */
+  public function setCurrency($currency) {
+    $this->currency = $currency;
+
+    return $this;
+  }
```

Comment:
You might consider whether all these getters and setters are actually needed. It might be possible to replace them with a proper constructor.

Reply:
Yes, that would be a possibility but not now.